manifest.yml

```diff
@@ -1,5 +1,9 @@
 name: netlify-plugin-lighthouse
 inputs:
+  - name: audits
```

Comment:
Moved ahead of its deprecated equivalent – we chould remove references to `audit_url` in the next major bump